cli/crates/federated-dev/src/dev/subgraph_config_watcher.rs

```diff
@@ -126,19 +126,21 @@ fn determine_changes<'a>(
     let deleted_names = previous_names.difference(&next_names).collect::<BTreeSet<_>>();
 
     let changed_subgraphs = {
-        let previous_subgraphs = previous_subgraphs
+        let previous_subgraph_names = previous_subgraphs
```

Comment:
This change is here because the config now has a float in it, so it can't implement `Ord` and `Eq`.